.gitignore

```diff
@@ -17,6 +17,8 @@ DerivedData
 *.ipa
 *.xcuserstate
 
+.DS_Store
```

Comment:
I recommend adding this to your global gitignore file, but it definitely makes sense in here too.